Sending and Receiving Coins Using your Skywallet.md

```diff
@@ -0,0 +1,100 @@
+# Sending and Receiving Coins Using your Skywallet
+
+Sending and receiving of Coins are two of the most basic features of your Skywallet.
+
+To send or receive Coins using your Skywallet, you have to initialize and personalize your Skywallet, details on how to initialize and personalize your Skywallet is explained in the following sections :
```

Comment:
..personalize your Skywallet first. Details on ... can be found in the following chapters: